modules/vba_documents/app/controllers/vba_documents/internal/v1/upload_complete_controller.rb

```diff
@@ -68,16 +68,23 @@ def process_upload(upload_id)
           store = VBADocuments::ObjectStore.new
           raise Common::Exceptions::RecordNotFound, upload_id unless upload && store.bucket.object(upload.guid).exists?
 
+          log_details = { 'job' => self.class.name }.merge(upload.as_json)
+
           if upload.status.eql?('pending')
-            Rails.logger.info("VBADocuments: Processing: #{upload.inspect}")
-            upload.update(status: 'uploaded')
+            Rails.logger.info("VBADocuments: Processing: #{upload.inspect}", log_details)
+
+            upload.update!(status: 'uploaded')
```

Comment:
This should have always been `update!` rather than `update`, so I've fixed it here (and in the `UploadScanner`). We don't want the update to fail silently.